rdmo/projects/assets/js/interview/utils/value.js

```diff
@@ -39,28 +39,33 @@ const initValues = (sets, values, element, setPrefix) => {
     setPrefix = ''
   }
 
+  // loop over all sets of the current set prefix and over all questions
   sets.filter((set) => set.set_prefix === setPrefix).forEach((set) => {
     element.elements.filter((e) => (e.model === 'questions.question')).forEach((question) => {
+      // check if there is any value for this question and set
       if (isNil(values.find((value) => (
         (value.attribute === question.attribute) &&
         (value.set_prefix == set.set_prefix) &&
         (value.set_index == set.set_index)
```

Comment:
do these need to be strict `===` as well?

Reply:
in principle `==` would be ok, I guess but I will change it (JS is weird).